media-gfx/zbar/zbar-0.23.93.ebuild

```diff
@@ -0,0 +1,231 @@
+	)"
+
+RDEPEND="${COMMON_DEPEND}
+	java? ( >=virtual/jre-1.8 )"
+
+DEPEND="${COMMON_DEPEND}
+	java? (
+		>=virtual/jdk-1.8
+		test? (
+			dev-java/hamcrest-core:1.3
+			dev-java/junit:4
+		)
+	)
+	test? (
+		${PYTHON_DEPS}
+		$(python_gen_cond_dep '
+			dev-python/pillow[${PYTHON_USEDEP}]
+		')
+		elibc_musl? ( sys-libs/argp-standalone )
+	)"
+
+BDEPEND="
+	app-text/xmlto
+	sys-devel/gettext
+	virtual/pkgconfig
+	gtk? ( dev-util/glib-utils )
+	nls? (
+		sys-devel/gettext
+		virtual/libiconv
+	)"
+
+DOCS=( README.md NEWS.md TODO.md HACKING.md TODO.md ChangeLog )
+
+PATCHES=(
+	# TODO: upstream?
+	"${FILESDIR}/${PN}-0.23_fix_python_detect.patch"
+	"${FILESDIR}/${PN}-0.23.90-fix-unittest.patch"
+)
+
+pkg_setup() {
+	if use python || use test; then
+		python-single-r1_pkg_setup
+	fi
+	use java && java-pkg-opt-2_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	if use python || use test; then
+		if use test; then
+			# make tests happy
+			# because one of the test requires loadable py module from the current ${BUILD_DIR}
+			sed -e "s|PYTHONPATH=@abs_top_srcdir@|PYTHONPATH=@builddir@|g" \
+				-i test/Makefile.am.inc || die
+		fi
+
+		python_fix_shebang \
+			examples/*.py \
+			test/{test_python,barcodetest}.py # test_pygtk.py — py2 only
+	fi
+
+	if use java; then
+		java-pkg-opt-2_src_prepare
+		sed -e "s|javadir = \$(pkgdatadir)|javadir = /usr/$(get_libdir)/zbar|" \
+			-i java/Makefile.am || die
+	fi
+
+	# do not install {LICENSE,INSTALL,etc} doc files with 'make install' (use DOCS=() instead)
+	sed -e "s|^dist_doc_DATA =\(.*\)|dist_doc_DATA =|" -i Makefile.am || die
+
+	eautoreconf
+}
+
+multilib_src_configure() {
+	append-cppflags -DNDEBUG
+
+	local myeconfargs=(
+		$(use_with dbus)
+		$(use_with gtk gtk gtk3) # avoid 'auto'
+		$(use_with jpeg)
+		$(multilib_native_use_with introspection gir)
+		$(multilib_native_use_with java)
+		$(multilib_native_use_with python python auto)
+		$(use_enable nls)
+		$(use_enable static-libs static)
+		$(use_enable threads pthread)
+		$(use_enable v4l video)
+		$(use_with X x)
+		$(use_with X xshm)
+		$(use_with xv xv)
+	)
+
+	if multilib_is_native_abi; then
+		# both must be enabled to use GraphicsMagick
+		if use imagemagick && use graphicsmagick; then
+			myeconfargs+=(
+				--with-graphicsmagick
+				--without-imagemagick
+			)
+		elif ! use graphicsmagick && use imagemagick; then
+			myeconfargs+=(
+				--with-imagemagick
+				--without-graphicsmagick
+			)
+		else
+			myeconfargs+=(
+				--without-imagemagick
+				--without-graphicsmagick
+			)
+		fi
+
+		if use java; then
+			export JAVACFLAGS="$(java-pkg_javac-args)"
+			append-cflags "$(java-pkg_get-jni-cflags)"
+			if use test; then # bug 629078
+				myeconfargs+=( --with-java-unit )
+				CLASSPATH+=":$(java-pkg_getjar --build-only junit-4 junit.jar)"
+				CLASSPATH+=":$(java-pkg_getjar --build-only hamcrest-core-1.3 hamcrest-core.jar)"
+			fi
+		fi
+
+		if use qt5; then
+			myeconfargs+=(
+				--with-qt
+				--with-qt5
```

Comment:
AFAIS there is no such configure option in 0.23.93. zbar defaults to QT5 and there is experimental support for QT6 (`--with-qt6`)

Reply:
Removed "qt5" option leaving "qt" only. Note there is no actual qt6 support in zbar, just a "prepare to support" commit to configure.ac which seems pointless before qt6 support is actually added to the code...